### PR TITLE
[jjo] add contrib/helpers.jsonnet, including unit tests

### PR DESCRIPTION
**Pull request by jjo**
See `contrib/helpers_usage_test.jsonnet` for example usage.

### CODE REVIEW

**Comment by arapulido** [comment]
Maybe would it be good to add some documentation of those to: https://github.com/bitnami/kube-prod-runtime/blob/master/docs/overrides.md

File: manifests/contrib/showtree.jsonnet [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  if std.objectHas(obj.metadata, 'namespace') then
+    { [k]: '%s %s -n %s' % [obj.kind, obj.metadata.name, obj.metadata.namespace] }
+  else
+    { [k]: '%s %s' % [obj.kind, obj.metadata.name] }
+);
+
+// Build main tree: jsonnet key -> formatted Kubernetes object
+local show_tree(obj) = (
+  local tree(key, x) = (
+    if std.objectHas(x, 'metadata') then
```
Comment (anguslees) [inline manifests/contrib/showtree.jsonnet]:
```suggestion
    if std.objectHas(x, 'apiVersion') && std.objectHas(x, 'kind') then
```

(checking both is probably unnecessarily strict)

**Comment by anguslees** [comment]
Merging.  We can think about if/how we promote/expose this later.

bors r+

**Pull request merged**